redirects.yaml

```diff
@@ -8,5 +8,35 @@ docs/(devel|2.6|stable|en|devel/en|stable/en|2.6/en)/(?P<path>.*): /docs/{path}
 docs/(?P<version>2.5|2.4|2.3|2.2|2.1|2.0|1.9)(/en)?/?: https://old-docs.maas.io/{version}/en/
 docs/(?P<version>2.5|2.4|2.3|2.2|2.1|2.0|1.9)(/en)?/(?P<path>.+): https://old-docs.maas.io/{version}/en/{path}
 
+legacy/images: http://images.maas.io
```

Comment:
Eeek, this broke - see #486 can we make this `legacy/images/.*` ? I don't know the syntax for redirects.yaml